### DISCUSSION TITLE
how reload for edge

### DISCUSSION THREAD

**Discussion by mkvcc** [adonisjs/core#2908] [Help] [answered]
Hello.
Is there a way to add hot reload for balde files? this works for encore related files but blade is not a part of that.
appreciate any help

**Comment by RomainLanz** [comment, marked as answer]
Hey @mkvcc! 👋 

Blade is not even part of AdonisJS since it is Laravel template engine.
We are using Edge file and they are cold-reloaded on save when using Webpack Encore.

**There is no way to use hot-reload with Edge file.**
